### PR TITLE
Create Hello Rails Back End

### PR DESCRIPTION
**Pull request by Debas-31**
# Hello Rails Back End
>In this project, implemented : 
- [x] Create a new Rails API app called 'hello-rails-back-end'.
- [x] Initialize your project with Git.
- [x] Make sure that your project has a Postgres database set up.
- [x] Create a table for storing your messages and create 5 different greetings.
- [x] Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and rails route).

## [Hello React Front End](https://github.com/Debas-31/hello-react-front-end)

### CODE REVIEW

**Comment by Kingstalux** [comment]
Hi @Debas-31, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request :x: 

- [ ] Kindly include the link to your backend repo in the README of your frontend repo :+1: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: README.md [hunk 1]
```diff
@@ -1,24 +1,75 @@
...
 
-* System dependencies
+- git clone the project
 
-* Configuration
+```bash 
+git clone git@github.com:Debas-31/hello-rails-back-end.git
 
-* Database creation
+cd rails-budget-app
```
Comment (TedLivist) [inline README.md]: - [ ] I don't we're building a budget-app, kindly fix the typo here.

File: README.md [hunk 2]
```diff
@@ -1,24 +1,75 @@
-# README
+# Hello Rails Back End
```
Comment (TedLivist) [inline README.md:1]: - [ ] I was unable to properly fetch greetings from the API due to CORS issue.

  err-cor

  You can fix this by installing the `rack-cors` gem and creating the `config/initializers/cors.rb` file and pasting the code below in it.

  ```
  Rails.application.config.middleware.insert_before 0, Rack::Cors do
    allow do
      origins '*'
      resource '*', headers: :any, methods: [:get, :post, :patch, :put]
    end
  end
  ```
  You can visit [this link for more](https://github.com/cyu/rack-cors)

**Review by TedLivist — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ ♻️

Hi @Debas-31,

Good job so far!
✔️ Good documentation
✔️ React app is well designed
However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. And please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (Debas-31) [reply README.md:1]: Thank you for the feedback. I will implement it!

**Review by addod19 — APPROVED** [review]
### Hi @Debas-31, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Highlights
- [ ] Nice job fixing the project name in the README file and also adding the rack-cors gem :100: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**